src/Plugin/EdgeKeyTypeBase.php

```diff
@@ -128,4 +149,16 @@ public function getClientSecret(KeyInterface $key): string {
     return $key->getKeyValues()['client_secret'] ?? Oauth::DEFAULT_CLIENT_SECRET;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getAccountKey(KeyInterface $key): array {
```

Comment:
@cnovak I think it would further complicate the user experience of setting up the connection by having a different key type for Hybrid, as it would involve exposing the key type on the module configuration page (admin/config/apigee-edge/settings), and then it would not be much different than setting up a key for use by the Edge module directly on the keys module settings (admin/config/system/keys/add).
I've added more sanitization upon saving the key so that only the relevant values are saved (ie, removing the unused values), so we can keep using this same class. Let me know if you'd like to discuss this further.

Reply:
Related question: Can we use the Apigee Edge Environment Variables key storage to store hybrid credentials? It can be used to store OAuth credentials and the token is stored elsewhere. Hybrid should work the same way and the environment variables should be quite similar that a user can use to store hyrid credentials (email, private key, auth server, etc).

Reply:
We may want to start using the environment variables key provider that is now part of the Key module instead. I see that `APIGEE_EDGE_ACCOUNT_JSON_KEY` exists in  Apigee Edge Environment Variables key provider, but having trouble getting formatting correct for key when stored as an environment variable in nginx.

Reply:
Hybrid implementation should also work with the Apigee Edge Environment Variables key storage. The `APIGEE_EDGE_ACCOUNT_JSON_KEY` environment var should be set to the JSON encoded account key - I haven't tried it in nginx, but in apache it works correctly if I just run the JSON key through a minifier to remove whitespaces and line breaks.